certified-connectors/ITautomate/apiDefinition.swagger.json

```diff
@@ -0,0 +1,272 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v1",
+    "title": "ITautomate",
+    "description": "ITautomate is an online platform that allows you to automate a wide variety of IT operational tasks, saving your business time, money and resources. Issue saved commands such as PowerShell cmdlets on remote devices and parse the results to use in future actions.",
+    "contact": {
+      "name": "ITautomate Support",
+      "url": "https://www.itautomate.io/contact-us/",
+      "email": "support@itautomate.io"
+    }
+  },
+  "host": "connectorapi.itautomate.io",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/RunCommand": {
+      "post": {
+        "tags": [
+          "DynamicCommand"
+        ],
+        "summary": "Issue PowerShell Command",
+        "description": "Issue a saved ITautomate PowerShell command to remote device(s)",
+        "operationId": "RunCommand",
+        "consumes": [
+          "application/json",
+          "application/x-www-form-urlencoded"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "description": "Issue a saved ITautomate PowerShell command to remote device(s)",
+            "in": "query",
+            "required": true,
+            "type": "integer",
+            "format": "int32",
+            "x-ms-summary": "id",
+            "x-ms-dynamic-values": {
+              "operationId": "ListAllCommands",
+              "parameters": {},
+              "value-collection": "Commands",
+              "value-path": "Id",
+              "value-title": "Name"
+            }
+          },
+          {
+            "name": "commandInput",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CommandInputParameter"
+            },
+            "x-ms-summary": "CommandInputParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CommandRunResultModel_X"
+            }
+          },
+          "default": {
+            "description": "OK",
```

Comment:
@ITautomate you can change it to something like `Error` if you want, you have `OK` for 200 code and for default right now.